bonus_guides/X_seeding_data.md

```diff
@@ -0,0 +1,53 @@
+When creating an app, it's important that we're able to seed our database with some initial data (e.g., for early development work or pre-launch testing purposes).
+
+Fortunately, Phoenix already provides us with a convention for seeding data. By default Phoenix generates a script file for each app at `priv/repo/seeds.exs`, which we can use to use to populate our database/repository.
+
+Also note that in order to seed data as in the example below you should have alrady generated and run the related migration (i.e., Link migration, controller, model, etc.) and updated your `router.ex`, as described in the [Ecto Models Guide](http://www.phoenixframework.org/docs/ecto-models) (if you haven't completed that Guide yet, should should do so before proceeding further).
+
+So in order to seed data, we simply need to add a script to `seeds.exs` that uses our database/repository to directly add the data we want. As you can see from the comments that Phoenix generated for us in `seeds.exs` file, we should follow this pattern:
```

Comment:
It might be more correct here to use "repository" instead of "database/repository". Thoughts?